bin/i18n/redact_restore_utils.rb

```diff
@@ -63,40 +63,33 @@ def self.restore_data(source_data, redacted_data, plugins = [], format = 'md')
     return JSON.parse(restored)
   end
 
+  # Restores redacted data into a json/yaml/yml file
+  #
+  # @param source [String] the source file path
+  # @param redacted [String] the redacted file path
+  # @param dest [String] the target (restored) file path
+  # @param plugins [Array] the list of plugins to use
+  # @param format [String] the format of the data (md, txt, etc.)
   def self.restore(source, redacted, dest, plugins = [], format = 'md')
-    return unless File.exist?(source)
-    return unless File.exist?(redacted)
-
-    if I18nScriptUtils.json_file?(source)
-      source_data = I18nScriptUtils.parse_file(source)
-      return unless source_data
-
-      redacted_data = I18nScriptUtils.parse_file(redacted)
-      return unless redacted_data
+    raise "[#{source}] source file does not exist" unless File.exist?(source)
+    raise "[#{redacted}] redacted file does not exist" unless File.exist?(redacted)
```

Comment:
Looks like the method will raise an exception now instead of just returning without a message. 👏

Reply:
Yes, it's better to know immediately that something is wrong than skip the restoration process

Reply:
I thought that it would be better to know immediately that something is wrong than skip the restoration process for the file without any notifications